### PR TITLE
updating mediation sdk and some adapters

### PR DESCRIPTION
**Pull request by imashok02**

### CODE REVIEW

File: build/providers/facebook/android/gradletealeaf.xml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 //<!-- Facebook -->
-  implementation 'com.ironsource.adapters:facebookadapter:4.3.2@jar'
-  implementation 'com.android.support:recyclerview-v7:25.3.1'
```
Comment (rampr) [inline build/providers/facebook/android/gradletealeaf.xml:3]: The build.gradle is also changing according to this https://developers.ironsrc.com/ironsource-mobile/android/mediation-networks-android/#step-1

Can you verify for others too

Comment (rampr) [reply build/providers/facebook/android/gradletealeaf.xml:3]: Also please verify if manifest has changes too

**Comment by rampr** [comment]
Have you checked if there were no changes needed in the manifest file?

**Comment by imashok02** [comment]
> Have you checked if there were no changes needed in the manifest file?

No changes in Manifest files

**Review by vivekb91 — APPROVED** [review]

**Review by rampr — APPROVED** [review]

**Pull request merged**